test/actor.js

```diff
@@ -1070,6 +1070,119 @@ describe('Apify.pushRecord()', () => {
             });
     });
 });
+describe('Apify.getOrCreateStore()', () => {
+    it('throws on invalid args', () => {
+        const keyErrMsg = 'The "storeName" parameter must be a non-empty string';
+        expect(() => { Apify.getOrCreateStore(); }).to.throw(Error, keyErrMsg);
+        expect(() => { Apify.getOrCreateStore('', null); }).to.throw(Error, keyErrMsg);
+        expect(() => { Apify.getOrCreateStore('', 'some value'); }).to.throw(Error, keyErrMsg);
+        expect(() => { Apify.getOrCreateStore({}, 'some value'); }).to.throw(Error, keyErrMsg);
+        expect(() => { Apify.getOrCreateStore(123, 'some value'); }).to.throw(Error, keyErrMsg);
+    });
+
+    it('returns a Promise', () => {
+        const promise = Apify.getOrCreateStore('my-store');
```

Comment:
Please also add a unit test to ensure that the function works with callbacks, not only promises.